src/System.Windows.Forms/src/System/Windows/Forms/PrintPreviewControl.PrintPreviewControlAccessibleObject.cs

```diff
@@ -25,5 +26,10 @@ public PrintPreviewControlAccessibleObject(PrintPreviewControl owner) : base(own
                 UiaCore.UIA.IsKeyboardFocusablePropertyId => (State & AccessibleStates.Focusable) == AccessibleStates.Focusable,
                 _ => base.GetPropertyValue(propertyID)
             };
+
+        public override Rectangle Bounds
+            => _owningPrintPreviewControl.IsHandleCreated && _owningPrintPreviewControl.Parent is not null
+                        ? _owningPrintPreviewControl.Parent.RectangleToScreen(_owningPrintPreviewControl.Bounds)
+                        : Rectangle.Empty;
```

Comment:
Please fix indentation:

```suggestion
            => _owningPrintPreviewControl.IsHandleCreated && _owningPrintPreviewControl.Parent is not null
                ? _owningPrintPreviewControl.Parent.RectangleToScreen(_owningPrintPreviewControl.Bounds)
                : Rectangle.Empty;
```